selfservice/flow/registration/flow.go

```diff
@@ -208,6 +202,10 @@ func (f *Flow) EnsureInternalContext() {
 	}
 }
 
+func (f *Flow) GetInternalContext() *sqlxx.JSONRawMessage {
+	return &f.InternalContext
```

Comment:
Reconsider pointer here - sqlxx.JSONRawMessage is a byte slice which means that modifying it will change the mem value and not create a copy (I think).

Reply:
We do want to change the internal context.

Reply:
fixed